services/server/src/server/controllers/verification/verification.common.ts

```diff
@@ -455,6 +456,12 @@ export const verifyContractsInSession = async (
     contractWrapper.status = getMatchStatus(match) || "error";
     contractWrapper.statusMessage = match.message;
     contractWrapper.storageTimestamp = match.storageTimestamp;
+    if (dryRun) {
+      logger.info("dryRun verification", {
+        sessionId: session.id,
```

Comment:
@kuzdogan for session cases, is it enough to log only the sessionId? All the other information are available in previous logs, right?

Reply:
Hmm it doesn't seem to be the case. You can see by running it once. But generally just log more things than less